source/extensions/extensions_metadata.yaml

```diff
@@ -3,6 +3,11 @@ envoy.access_loggers.file:
   - envoy.access_loggers
   security_posture: robust_to_untrusted_downstream
   status: stable
+envoy.access_loggers.filters.cel:
+  categories:
+  - envoy.access_loggers.filters
```

Comment:
there isnt currently a category for this - not sure if this should just be `envoy.access_loggers` or whether there is a case for adding such a category

Reply:
I wasn't sure either. But just adding it in `envoy.access_loggers` didn't quite seem right either (what if there is a future access logger that uses CEL expressions for the log format, or similar?). Don't have strong feelings tho, so happy to update as makes sense.

Reply:
yep, that makes sense to me

extension "categories" in this context are kinda the extension points, so there would be some additional code overhead setting that up (beyond my immediate knowledge on how to do it im afraid)

Reply:
`envoy.access_loggers.extension_filters` here too